Coding/fw/src/main.cpp

```diff
@@ -51,4 +68,20 @@ void loop()
 
 /* EXTERNAL FUNCTIONS *****************************************************************************/
 
-/* INTERNAL FUNCTIONS *****************************************************************************/
\ No newline at end of file
+/* INTERNAL FUNCTIONS *****************************************************************************/
+
+void restart()
+{
+  delay(5000);
+  protocolLawicel.end();
+  ESPServer::end();
+  Board::reset();
+}
+
+void setErrorLED()
```

Comment:
guess you shold call it errorHalt() as the must intrusive part of it is the endless loop.

Isn't there a hlt() instruction ? To save power, a halt loop looks typically like this:
while(true)
{
    hlt();
}

somethimes even interrrupts get turnes off (cli());